functions/src/Services/Event.ts

```diff
@@ -35,30 +46,40 @@ export const rsvp = (event: Event, user: User) => {
 	// App.database().ref(`events/${event.id}/rsvp`).update({ [user.id]: !(user.id in event.rsvp) });
 };
 
+/**
+ * userReference and eventReference do not use their respective converters, because they're stored as documentReferences in the firestore
+ *
+ * The attendees subCollection inside of an event stores documentReferences to users that attended that event,
+ * that way if any of the user data updates, we don't have to update the subCollection inside of an event
+ *
+ * The eventsAttended subCollection works the same way as the attendees subCollection, except this one stores references to events attended by the user
+ */
 export const checkIn = (event: Event, user: User, showAlert = true) => {
+	const rsvpBonus = event.rsvp && user.id in event.rsvp ? 1 : 0;
```

Comment:
Wrap condition in parenthesis
```suggestion
	const rsvpBonus = (event.rsvp && user.id in event.rsvp) ? 1 : 0;
```